google-cloud-logging/lib/google/cloud/logging/async_writer.rb

```diff
@@ -146,12 +153,14 @@ def initialize logging, max_queue_size = DEFAULT_MAX_QUEUE_SIZE
         #
         #   async.write_entries entry
         #
-        def write_entries entries, log_name: nil, resource: nil, labels: nil
+        def write_entries entries, log_name: nil, resource: nil, labels: nil,
+                          partial_success: nil
```

Comment:
I believe this argument should go on `Project#async_writer`, not `AsyncWriter#write_entries`

Reply:
Ah I see. Just changed `partial_success` to be an instance variable.